UniTrain/utils/classification.py

```diff
@@ -1,14 +1,37 @@
 import os
-import UniTrain
 from torch.utils.data import DataLoader
 from torchvision import transforms
-from UniTrain.dataset.classification import ClassificationDataset
+from ..dataset.classification import ClassificationDataset
```

Comment:
You would have to 
import UniTrain
from UniTrain.dataset.classification import ClassificationDataset

.dataset.classification would give error